web/testing/issue3701/test3701/source/welcome.htm

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Start Using test3701</title>
+  <style type="text/css">
+    p { font: 10pt Tahoma; }
+    h1 { font: bold 16pt Tahoma; color: #4444cc; margin-bottom: 2px }
+    h2 { font: bold 12pt Tahoma; color: #4444cc; }
+  </style>
+</head>
+<body>
+
+<h1>Start Using test3701</h1>
+
+<p>
+    test3701 1.0 generated from template.

```

Comment:
Even though this is from template, maybe add a description that the keyboard prohibits use of the letter <kbd>x</kbd> (and <kbd>X</kbd>)?